### PR TITLE
Support for setting up users git config (user.name/user.email)

### PR DESCRIPTION
**Pull request by sagar-gavhane**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation

## Description

Support for setting up users git config

## Related Tickets & Documents

#1 

## Added tests?

- [ ] Yes
- [x] No, because they aren't needed
- [ ] No, because I need help

## Does this introduce a breaking change?

- [x] Yes
- [ ] No

### CODE REVIEW

**Comment by sagar-gavhane** [comment]
@ahmadawais any updates on this PR?

File: action.yml [hunk 1]
```diff
@@ -27,6 +27,14 @@ inputs:
...
         required: false
         default: '.'
+    user_name:
+        description: 'Determine name of git committer'
+        required: false
+        default: 'Ahmad Awais'
+    user_email:
+        description: 'Determine email of git committer'
+        required: false
+        default: 'me@ahmadawais.com'
```
Comment (ahmadawais) [inline action.yml]: I would never want my email here. Please remove this.

File: action.yml [hunk 2]
```diff
@@ -27,6 +27,14 @@ inputs:
         description: 'Directory to change to before pushing.'
         required: false
         default: '.'
+    user_name:
+        description: 'Determine name of git committer'
+        required: false
+        default: 'Ahmad Awais'
```
Comment (ahmadawais) [inline action.yml]: Please remove my name from here.

**Review by ahmadawais — COMMENTED** [review]
Looks pretty good. Make the changes I requested and I'll merge this one.

**Review by ahmadawais — CHANGES_REQUESTED** [review]
Check the comments.

**Comment by sagar-gavhane** [comment]
Comments resolved 👍

**Pull request closed without merging**